test/mocha.opts

```diff
@@ -1,2 +1,3 @@
 --full-trace
 --reporter dot
+--globals Base64
```

Comment:
That's weird, why is this needed?

Reply:
I couldn't figure out where the heck it is coming from. I can try and dig in more. Would it come from a dependent package?! That is the only thing I can think of.

Reply:
Yep, looks like a dependency does `Base64 = ...` instead of `var Base64 = ...`. Could you check where this is coming from? (search for `Base64` in `node_modules`) And send a PR to the broken package. I can't accept this change.

Reply:
Caused by https://github.com/dankogai/js-base64/commit/39b64ba0316f71f438cacab4b821b1947cd0f491

cc @dankogai

Reply:
`postcss@5` depends on `js-base64`.